modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -27,59 +29,68 @@ Skipping this step causes the second boot to fail.
 ----
 
 [start=5]
-. Generate a thumbprint from the Tang server.
-
-.. In the following command, replace the value of `url` with the Tang server URL:
+. On the {op-system-base} 8 machine, run the following command to generate a thumbprint of the exchange key. Replace `\http://tang.example.com:7500` with the URL of your Tang server:
 +
 [source,terminal]
 ----
-$ echo nifty random wordwords \
-     | clevis-encrypt-tang \
-       '{"url":"https://tang.example.org"}'
+$ clevis-encrypt-tang '{"url":"http://tang.example.com:7500"}' < /dev/null > /dev/null <1>
 ----
+<1> In this example, `tangd.socket` is listening on port `7500` on the Tang server.
++
+[NOTE]
+====
+The `clevis-encrypt-tang` command is used in this step only to generate a thumbprint of the exchange key. No data is being passed to the command for encryption at this point, so `/dev/null` is provided as an input instead of plain text. The encrypted output is also sent to `/dev/null`, because it is not required for this procedure.
+====
 +
 .Example output
 [source,terminal]
 ----
 The advertisement contains the following signing keys:
 
-PLjNyRdGw03zlRoGjQYMahSZGu9
+PLjNyRdGw03zlRoGjQYMahSZGu9 <1>
 ----
+<1> The thumbprint of the exchange key.
++
+When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`.
++
+[NOTE]
+====
+{op-system-base} 8 provides Clevis version 15, which uses the SHA-1 hash algorithm to generate thumbprints. Some other distributions provide Clevis version 17 or later, which use the SHA-256 hash algorithm for thumbprints. You must use a Clevis version that uses SHA-1 to create the thumbprint, to prevent Clevis binding issues when you install {op-system-first} on your {product-title} cluster nodes.
+====
 
-.. When the `Do you wish to trust these keys? [ynYN]` prompt displays, type `Y`, and the thumbprint is displayed:
+. If you have not yet generated the Kubernetes manifests, change to the directory that contains the installation program on your installation node and create them:
 +
-.Example output
 [source,terminal]
 ----
-eyJhbmc3SlRyMXpPenc3ajhEQ01tZVJiTi1oM...
+$ ./openshift-install create manifests --dir=<installation_directory> <1>
 ----
+<1> Replace `<installation_directory>` with the path to the directory that you want to store the installation files in.
 
-. In the `openshift` directory, create master or worker files to encrypt disks for those nodes.
+. Create machine config files to encrypt the boot disks for the control plane or compute nodes using the Tang encryption mode. 
 +
 [IMPORTANT]
 ====
-If you are also configuring boot disk mirroring on the affected nodes, record the server thumbprint for later use, and skip this step. You will configure disk encryption when configuring boot disk mirroring.
+If you are also configuring boot disk mirroring on the affected nodes, record the exchange key thumbprint for later use, and skip this step. You will configure disk encryption when configuring boot disk mirroring.
```

Comment:
In the following `Mirroring disks during installation` section, also has some reference about `https://tang.example.com`, also need to correct.

Reply:
Thank you for highlighting that. I have now updated that reference in the mirroring section.